From d101895a1ddebce9cf57b5f8dac6511c07452d57 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 31 Aug 2023 07:49:39 +0000
Subject: [PATCH] Enable drm aidl service

Android U only supports aidl for drm. Enabling
google default aidl drm service

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/default-drm/true/product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/default-drm/true/product.mk b/groups/default-drm/true/product.mk
index 9ebe493..f32c285 100644
--- a/groups/default-drm/true/product.mk
+++ b/groups/default-drm/true/product.mk
@@ -1,3 +1,3 @@
 #only enable default drm service
-PRODUCT_PACKAGES += android.hardware.drm@1.4-service.clearkey
+PRODUCT_PACKAGES += android.hardware.drm-service.clearkey
 
-- 
2.39.2

